projects/noloan/app/src/main/java/hasadna/noloan/InboxRecyclerAdapter.java

```diff
@@ -75,41 +79,25 @@ public void bind(SmsMessage sms) {
       content.setText(sms.getBody());
 
       // Search if message was suggested / user suggested: update counter / add undo button
-      if (SmsMessages.get().searchDbMessage(sms) != -1) {
-        counter.setText(
-            itemView
-                .getResources()
-                .getString(
-                    R.string.list_item_textView_spam_counter,
-                    SmsMessages.get()
-                        .getDbMessages()
-                        .get(SmsMessages.get().searchDbMessage(sms))
-                        .getSuggestersCount()));
+      DbMessages dbMessages = DbMessages.getInstance();
+      int index = DbMessages.findSms(sms, dbMessages.getMessages());
```

Comment:
You can create static methods like `getMessages()` that call `DbMessages.getInstance().getMessages()` so you don't have to use `DbMessages dbMessages = DbMessages.getInstance();` every time, and instead just do `DbMessages.getMessages()` for example.